src/js/formats/helpers/buildCarousel.js

```diff
@@ -59,8 +59,8 @@ export default function buildCarousel(length) {
 
         q('.js-indicator').forEach((el) => {
             el.addEventListener('click', () => {
-                const elementId = el.id || el.parentNode.id;
-                const newSlideIndex = elementId.split(/carousel-indicator-/)[1];
+                const newSlideIndex = (el.dataset && el.dataset.slideIndex) ||
```

Comment:
I know, right! MDN just told me about it. IE11 and above, fine for our needs. The worst that can happen in older browsers is that clicking the carousel indicators doesn't do anything.